sql/_13_issues/_24_1h/answers/cbrd_25277.answer

```diff
@@ -0,0 +1,221 @@
+===================================================
+test data    
+     
+
+Query plan:
+sscan
+    class: dual node[?]
+    cost:  ? card ?
+Query stmt:
+select '' from dual dual
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+1
+===================================================
+1
+===================================================
+1
+===================================================
+1
+===================================================
+1
+===================================================
+0
+===================================================
+0
+===================================================
+0
+===================================================
+test case 1    
+AS-IS - Outer join query optimization failed.
+TO-BE
+- Eliminate INNER JOIN for the first table
+- INNER JOIN
+- With ANSI JOIN syntax
+     
+
+Query plan:
+sscan
+    class: dual node[?]
+    cost:  ? card ?
+Query stmt:
+select 'AS-IS - Outer join query optimization failed.
+===================================================
+c_b    c_b    
+2     20     
+
+Query plan:
+idx-join (inner join)
+    outer: sscan
+               class: c node[?]
+               cost:  ? card ?
+    inner: iscan
+               class: d node[?]
+               index: pk_tc_dummy_c_a term[?]
+               cost:  ? card ?
+    cost:  ? card ?
+Query stmt:
+select /*+ ORDERED */ c.c_b, d.c_b from  inner join tb_child c inner join tc_dummy d on d.c_a=c.c_a
```

Comment:
There is an error in the rewritten query.
The "inner join" should be removed.
If rewritten as above, it cannot be processed properly in plcsql.
Please report to the development team.

Reply:
@youngjinj 
when executing a rewritten query directly occurs syntax error as below:
Syntax error: unexpected 'inner'

It will not be a problem now but if use same query in PL/CSQL it will not be executed as expected.

Reply:
I hope this will be discussed in another issue.

Reply:
@kwonhoil, @tw-kang
The issue of needing to remove the 'inner join' keyword will be resolved in a new PR for the same issue.